src/plugins/simulator/fsxcommon/simconnectdatadefinition.cpp

```diff
@@ -424,6 +425,23 @@ namespace swift::simplugin::fsxcommon
         return hr;
     }
 
+    // TODO TZ: are there any changes in MSFS2024 ?
```

Comment:
At least from the docs, nothing has changed:

- https://docs.flightsimulator.com/msfs2024/html/6_Programming_APIs/SimVars/Aircraft_SimVars/Aircraft_RadioNavigation_Variables.htm#TRANSPONDER_STATE
- https://docs.flightsimulator.com/msfs2024/html/6_Programming_APIs/SimVars/Aircraft_SimVars/Aircraft_RadioNavigation_Variables.htm#TRANSPONDER_IDENT

Reply:
Comment removed